docs/CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+
+## 1.0.0b1
+
+Feature:
+* Get configs via Airflow Configuration manager [#503](https://github.com/astronomer/astro-sdk/issues/503)
+* Optimize loading file directly from GCS to Bigquery via load_file operator [#429](https://github.com/astronomer/astro-sdk/issues/429)
+* Optimize loading file directly from S3 to Bigquery via load_file operator [#429](https://github.com/astronomer/astro-sdk/issues/429)
+* Optimize loading file directly from local to bigquery via load_file operator [#429](https://github.com/astronomer/astro-sdk/issues/429)
+* Optimize performance loading file to postgres via load_file operator [#428](https://github.com/astronomer/astro-sdk/issues/428)
+* Optimize loading file directly from GCS to snowflake via load_file operator [#430](https://github.com/astronomer/astro-sdk/issues/430)
+* Optimize loading file directly from S3 to snowflake via load_file operator [#430](https://github.com/astronomer/astro-sdk/issues/430)
```

Comment:
I believe these are enhancements, not new features - so I think they should be placed in the section "Enhancements", similar to release 0.11.0.

From a user perspective, I believe what matters is to state (referencing any tickets associated to each of these improvements, including running benchmarks before/after):
* Improved the performance of `aql.load_file` for files from S3 to BQ up to X%.
* Improved the performance of `aql.load_file` for files from GCS to BQ up to X%.
* Improved the performance of `aql.load_file` for files from S3/GCS to Snowflake up to X%.
* Improved the performance of `aql.load_file` for files to Postgres up to X%.

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/576/commits/88b0e130c53839808f7fa7be3df528526f57e8ab)